pkg/scheduler/core/pod_scheduler/pod_scheduler_test.go

```diff
@@ -61,6 +65,9 @@ func TestSelectHost(t *testing.T) {
 			},
 			possibleHosts: sets.NewString("machine2.1"),
 			expectsErr:    false,
+			pod: testinghelper.MakePod().Namespace("default").Name("pod1").UID("pod1").
+				ControllerRef(metav1.OwnerReference{Kind: "ReplicaSet", Name: "rs1", UID: "rs1"}).Obj(),
+			expectedCacheSizes: []int{1, 2, 3, 4, 5, 6, 7, 8, 9, 10},
```

Comment:
This is a bit strange, under the current test logic, the same node will be cached repeatedly (x, x\*2, x\*3, ... depending on the number of tests at #L118). I tend to put the initialization of 'podScheduler' at the beginning of each test case to avoid subsequent effects, and 'expectedCacheSizes' is no longer needed.

Reply:
thanks for your feedback, i will modify it.